src/fiber/fiber.mli

```diff
@@ -224,6 +224,30 @@ module Mutex : sig
 end
 with type 'a fiber := 'a t
 
+module Throttle : sig
+  (** Limit the number of jobs *)
+
+  type 'a fiber = 'a t
+
+  type t
+
+  (** [create n] creates a throttler that allows to run [n] jobs at once *)
+  val create : int -> t
+
+  (** How many jobs can run at the same time *)
+  val size : t -> int
+
+  (** Change the number of jobs that can run at once *)
+  val resize : t -> int -> unit fiber
+
+  (** Execute a fiber, waiting if too many jobs are already running *)
+  val run : t -> (unit -> 'a fiber) -> 'a fiber
```

Comment:
Perhaps we should make the function argument labeled with `f` for consistency?

Reply:
Seems good